src/Protocol/PortableContact.php

```diff
@@ -1647,27 +1647,33 @@ public static function discover($complete = false)
 
 		$no_of_queries = 5;
 
-		$requery_days = intval(Config::get("system", "poco_requery_days"));
+		$requery_days = intval(Config::get('system', 'poco_requery_days'));
 
 		if ($requery_days == 0) {
 			$requery_days = 7;
 		}
-		$last_update = date("c", time() - (60 * 60 * 24 * $requery_days));
+		$last_update = date('c', time() - (60 * 60 * 24 * $requery_days));
 
-		$r = q("SELECT `id`, `url`, `nurl`, `network` FROM `gserver` WHERE `last_contact` >= `last_failure` AND `poco` != '' AND `last_poco_query` < '%s' ORDER BY RAND()", DBA::escape($last_update));
-		if (DBA::isResult($r)) {
-			foreach ($r as $server) {
-				if (!self::checkServer($server["url"], $server["network"])) {
+		$gservers = q("SELECT `id`, `url`, `nurl`, `network`
+			FROM `gserver`
+			WHERE `last_contact` >= `last_failure`
+			AND `poco` != ''
+			AND `last_poco_query` < '%s'
+			ORDER BY RAND()", DBA::escape($last_update)
```

Comment:
I guess we don't need the ```RAND()``` here. This means that in some future step we can replace this with some ```dba::select```.

Reply:
What about the `>=` and `<` operators?

Reply:
The condition can be something like this: 
```$condition = "[`last_contact` >= `last_failure` AND `poco` != '' AND `last_poco_query` < ?", $last_update]```